packages/dashboard/src/components/CompositeList/CompositeList.tsx

```diff
@@ -0,0 +1,230 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import * as stickyfill from 'stickyfilljs';
+
+import { CommandBar, ICommandBarItemProps } from 'office-ui-fabric-react/lib/CommandBar';
+import {
+  ConstrainMode,
+  DetailsListLayoutMode,
+  IColumn as ICompositeListColumn,
+  IDetailsList,
+  IDetailsRowProps
+} from 'office-ui-fabric-react/lib/DetailsList';
+import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
+import { ISelection, Selection, SelectionMode } from 'office-ui-fabric-react/lib/Selection';
+import { ShimmeredDetailsList } from 'office-ui-fabric-react/lib/ShimmeredDetailsList';
+import { mergeStyles, mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
+import { IRenderFunction } from 'office-ui-fabric-react/lib/Utilities';
+
+import {
+  getCommandBarItemStyle,
+  getCommandBarSearchBoxStyle,
+  getCommandBarSelectionCountItemStyles,
+  getCommandBarStyle,
+  getCommandBarWrapperStyle,
+  getDetailsListStyle
+} from './CompositeList.styles';
+
+import { ICompositeListFarItemOptions, ICompositeListProps } from './CompositeList.types';
+
+export interface ICompositeListStates {
+  numberOfItemsSelected: number;
+  isCompactMode: boolean;
+}
+
+export class CompositeList extends React.PureComponent<ICompositeListProps, ICompositeListStates> {
+  private readonly selection: ISelection;
+  private readonly selectionMode: SelectionMode;
+  private readonly commandBarRef = React.createRef<HTMLDivElement>();
+  private readonly detailsListRef = React.createRef<IDetailsList>();
+
+  constructor(props: ICompositeListProps) {
+    super(props);
+
+    this.selectionMode = props.detailsListProps.selectionMode || SelectionMode.multiple;
+
+    this.selection =
+      this.props.detailsListProps.selection ||
+      new Selection({
+        onSelectionChanged: () => {
+          this.setState({
+            numberOfItemsSelected: this.selection.getSelectedCount()
+          });
+        },
+        selectionMode: this.selectionMode
+      });
+
+    this.state = {
+      numberOfItemsSelected: 0,
+      isCompactMode: false
+    };
+  }
+  public render(): React.ReactNode {
+    const wrappedColumns =
+      this.props.detailsListProps.columns === undefined
+        ? undefined
+        : this.props.detailsListProps.columns.map((column: ICompositeListColumn) => {
+            return {
+              isResizable: true,
+              ...column
+            };
+          });
+
+    const useReducedRowRenderer =
+      wrappedColumns === undefined || !wrappedColumns.some((column: ICompositeListColumn) => column.isResizable === true);
```

Comment:
>column.isResizable === true [](start = 93, length = 27)

column.isResizable should be enough, do we need the explicit === true?

Reply:
I think so.